### DISCUSSION TITLE
Log received frame size in WebSocketFrameDecoder

### DISCUSSION THREAD

**Discussion by rishabhjainps** [netty/netty#13886] [Q&A]
Hi we are trying the log received frame size which is exceeding max size in WebSocket08FrameDecoder. Currently we don't have a way to log the receive frame size without actually decoding the frame here -   Payload length:  7 bits - https://datatracker.ietf.org/doc/html/rfc6455#section-5.2

Is there recommended we can log received frame size which is erroring out with CorruptedWebSocketFrameException or we can add frame size exceeded in message in below code ?

1.  [WebSocket08FrameDecoder code link] (https://github.com/netty/netty/blob/85cee9d1a8881a93e7c43a02baefea75b6ade2b5/codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocket08FrameDecoder.java#L289)

**Comment by normanmaurer** [comment]
@rishabhjainps feel free to open a PR to include the received frame size in the message.
